mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -3,18 +3,26 @@ CLASS net/minecraft/unmapped/C_zpccxxiz net/minecraft/util/math/MathHelper
 	FIELD f_aomqjoqj RADIANS_PER_DEGREE F
 	FIELD f_bciswahe DEGREES_TO_SINE_TABLE_INDEX F
 	FIELD f_buotfbwr SINE_TABLE [F
-	FIELD f_clwgxcor HALF_PI_RADIANS_SINE_TABLE_INDEX J
```

Comment:
oh, i'm actually curious about this field; does the new name match it's current usages?

Reply:
as far as I've found, its only usage is in `randomUuid`, so yes
the reason for the incorrect yarn name is because a float of the same value is used in `cos`

Reply:
ohh, I see; In that case? *pats mojmap*